models/semantic_segmentation/DUC/README.md

```diff
@@ -0,0 +1,72 @@
+# DUC
+
+## Use cases
+DUC is a semantic segmentation model, i.e., for an input image the model labels each pixel in the image into a set of pre-defined categories. The model provides very good accuracy in terms of mIOU score and can be used in any application requiring semantic segmentation. In particular, since the model is trained on the [cityscapes dataset](#dset) which contains images from urban street scenes, it can be used effectively in self driving vehicle systems.
```

Comment:
Add a section or note describing what is mIOU score

Reply:
linked to section below